cmd/gossamer/configcmd_test.go

```diff
@@ -229,30 +213,24 @@ func TestSetGlobalConfig(t *testing.T) {
 		c := c // bypass scopelint false positive
 		t.Run(c.description, func(t *testing.T) {
 			context, err := createCliContext(c.description, c.flags, c.values)
-			if err != nil {
-				t.Fatal(err)
-			}
+			require.Nil(t, err)
 
 			tCfg := &cfg.GlobalConfig{}
 
 			setGlobalConfig(context, tCfg)
 
-			if !reflect.DeepEqual(*tCfg, c.expected) {
-				t.Errorf("\ngot: %+v \nexpected: %+v", tCfg, c.expected)
-			}
+			require.Equal(t, c.expected, *tCfg)
 		})
 	}
 }
 
 func TestCreateP2PService(t *testing.T) {
 	gendata := &genesis.GenesisData{
-		ProtocolId: "gossamer",
+		ProtocolID: "gossamer",
```

Comment:
Can we please change this to `/gossamer/test` to match the protocol id in other tests?

Reply:
I dont understand your request

Reply:
```
ProtocolID: "/gossamer/test"
```

Reply:
sure